src/system/WatchableEventManagerSelect.cpp

```diff
@@ -87,6 +88,7 @@ void WatchableEventManager::Signal()
     {
         ChipLogError(chipSystemLayer, "System wake event notify failed: %" CHIP_ERROR_FORMAT, ChipError::FormatError(status));
```

Comment:
I'm not sure if I agree with this one being able to fail either..

If it fails, and all of the possible failures are either catastrophic faults or failed preconditions, do we really need a return value versus adding an assertion?